packages/react-charts/src/components/ChartLegend/examples/ChartLegend.md

```diff
@@ -680,7 +680,7 @@ class LegendLinkPieChart extends React.Component {
         <Chart
           ariaDesc="Average number of pets"
           ariaTitle="Line chart example"
-          containerComponent={<ChartVoronoiContainer labels={({ datum }) => `${datum.name}: ${datum.y}`} constrainToVisibleArea />}
+          containerComponent={<ChartContainer role="group" labels={({ datum }) => `${datum.name}: ${datum.y}`} constrainToVisibleArea />}
```

Comment:
This needs to be `ChartVoronoiContainer` to show tooltip values related to x and y coordinates.

In addition, I envisioned adding `role="group"` as a default.

Reply:
It appears that Victory uses role="img" as the default to describe the SVG element